vm/src/stdlib/os.rs

```diff
@@ -1268,6 +1270,20 @@ mod _os {
         env::set_current_dir(&path.path).map_err(|err| err.into_pyexception(vm))
     }
 
+    #[cfg(not(windows))]
+    #[pyfunction]
+    fn fchdir(fd: PyIntRef, vm: &VirtualMachine) -> PyResult<()> {
```

Comment:
this function need to be place in `posix` module. (try `import posix; posix.fchdir` in CPython`)
Then it will be automatically imported in os module.
Then `#[cfg(not(windows))]` is also not required anymore.

Reply:
That's a great point! I'll push the refined one.